GPU/GPUTracking/Base/cuda/GPUReconstructionCUDA.cu

```diff
@@ -37,6 +37,10 @@
 #undef GPUCA_KRNL
 #endif
 
+#ifdef GPUCA_HAS_ONNX
```

Comment:
> Why do you need the GPU specifix settings for the general GPUTracking library? Can't you put them only into the backend libraries?

I need the definition of GPUCA_HAS_ONNX here. and the ORT variables further below in the same file